extensions/libraries/redcore/form/fields/rdatepicker.php

```diff
@@ -77,13 +230,95 @@ protected function getInput()
 	}
 
 	/**
+	 * Matches each symbol of PHP date format standard
+	 * with jQuery equivalent codeword
+	 *
+	 * @param   string  $jQueryFormat  $jQueryFormat date format
+	 *
+	 * @author Tristan Jahier
+	 *
+	 * @return  string
+	 */
+	protected function dateFormatJQueryUIToPHP($jQueryFormat)
+	{
+		$phpFormat = "";
+		$escaping = false;
+		$length = strlen($jQueryFormat);
+
+		for ($i = 0; $i < $length; $i++)
+		{
+			$char = $jQueryFormat[$i];
+
+			// JQuery date format escaping character
+			if ($char === '\'')
+			{
+				if ($escaping)
+				{
+					$escaping = false;
+				}
+				else
+				{
+					$i++;
+					$phpFormat .= '\\' . $jQueryFormat[$i];
+					$escaping = true;
+				}
+			}
+			else
+			{
+				if ($escaping || !($findMatch = $this->findMatch(substr($jQueryFormat, $i))))
+				{
+					if ($char == ' ')
+					{
+						$phpFormat .= $char;
+					}
+					else
+					{
+						$phpFormat .= "\\" . $char;
+					}
+				}
+				else
+				{
+					$phpFormat .= $this->symbolsMatching[$findMatch];
+					$i += strlen($findMatch) - 1;
+				}
+			}
+		}
+
+		return $phpFormat;
+	}
+
+	/**
+	 * Find match date template
+	 *
+	 * @param   string  $string  Current string
+	 *
+	 * @return bool|string
+	 */
+	protected function findMatch($string)
+	{
+		$length = 0;
+		$itemKey = false;
+
+		foreach ($this->symbolsMatching as $key => $item)
+		{
+			if (strpos($string, $key) === 0 && $length < strlen($item))
+			{
+				$length = strlen($item);
+				$itemKey = $key;
+			}
+		}
+
+		return $itemKey;
+	}
+
+	/**
 	 * Build the datepicker JS options
 	 *
 	 * @return  string  Options in json format
 	 */
 	protected function getDatepickerOptions()
 	{
-		$options = new JRegistry;
+		$return = '{';
```

Comment:
I didn't get you. What don't you see? There is https://github.com/redCOMPONENT-COM/redCORE/pull/744/files#diff-539a740304d8b474cac4672d6cb24bb5R395 an example where we can set function instead of text.